docs/firewall_opnsense.rst

```diff
@@ -0,0 +1,780 @@
+
+In the **Generic configuration** section, select ``Static IPv4`` in the **IPv4
+Configuration Type** dropdown, and ``None`` in the **IPV6 Configuration Type**
+dropdown.
+
+Scroll down. In the **Static IPv4 Configuration** section, enter the *Application
+Gateway* IP address and routing prefix (``10.20.2.1`` and ``24`` if you are using
+the recommended values).
+
+Click **Save**, then click **Apply changes** when prompted.
+
+Configure the OPT2 interface
+'''''''''''''''''''''''''''''
+Finally, navigate to **Interfaces > [OPT2]**. In the **Basic configuration** section,
+check the checkbox labeled **Prevent interface removal**.
+
+In the **Generic configuration** section, select ``Static IPv4`` in the **IPv4
+Configuration Type** dropdown, and ``None`` in the **IPV6 Configuration Type**
+dropdown.
+
+Scroll down. In the **Static IPv4 Configuration** section, enter the *Monitor
+Gateway* IP address and routing prefix (``10.20.3.1`` and ``24`` if you are using
+the recommended values).
+
+Click **Save**, then click **Apply changes** when prompted.
+
+Configure Firewall Aliases
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+In order to simplify firewall rule setup, the next step is to configure aliases
+for hosts and ports referred to in the rules.
+
+To start, first navigate to **Firewall > Aliases**. You should see some system-defined
+aliases as shown below:
+
+|OPNSense - Alias Start|
+
+Click the **+** button to add new aliases. You should add the aliases defined in
+the table below (assuming recommended values for IP addresses):
+
+.. list-table:: Firewall Aliases
+   :header-rows: 1
+
+   * - Name
+     - Type
+     - Content
+
+   * - admin_workstation
+     - Host(s)
+     - 10.20.1.2
+
+   * - app_server
+     - Host(s)
+     - 10.20.2.2
+
+   * - external_dns_servers
+     - Host(s)
+     - 8.8.8.8, 8.8.4.4
+
+   * - monitor_server
+     - Host(s)
+     - 10.20.3.2
+
+   * - local_servers
+     - Host(s)
+     - app_server, monitor_server
+
+   * - OSSEC
+     - Port(s)
+     - 1514
+
+   * - ossec_agent_auth
+     - Port(s)
+     - 1515
+
+   * - antilockout_ports
+     - Port(s)
+     - 80, 443
+
+When complete, the **Aliases** page should look like this:
+
+|OPNSense - aliases end|
+
+Scroll down and click **Apply** to save and apply your new aliases.
+
+Configure Firewall Rules
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+Next, configure firewall rules for each interface.
+
+
+Configure Firewall Rules on LAN
+'''''''''''''''''''''''''''''''
+First, navigate to **Firewall > Rules > LAN**.  The LAN interface should have one
+automatically-generated anti-lockout rule in place, in addition to two default-allow rules.
+The default-allow rules should be removed once the SecureDrop-specific rules below
+have been added. The anti-lockout feature should be disabled as a last step.
+
+The rules needed are described in this table:
+
+.. list-table:: Firewall Rules - LAN
+   :header-rows: 1
+
+   * - Action
+     - Protocol
```

Comment:
I show OPNsense distinguishing the *TCP/IP Version* and **Protocol** fields:

![image](https://user-images.githubusercontent.com/357435/142338157-341dfbb5-fe76-4be4-98ba-f64b19ac63a6.png)

Reply:
Yup, I stuck those together in the table just to try to keep it concise, but at the very least the column header should reflect that. I'm gonna split it out to two columns, just because otherwise the width difference between header and content cells will drive me crazy :)